src/main/java/org/apache/sling/servlets/post/AbstractPostOperation.java

```diff
@@ -62,7 +66,7 @@ public abstract class AbstractPostOperation implements PostOperation {
     /**
      * default log
      */
-    protected final Logger log = LoggerFactory.getLogger(getClass());
+    protected final Logger log = LoggerFactory.getLogger(AbstractPostOperation.class);
```

Comment:
Why this change? now all subclasses log with the same logger, instead of using separate ones